Earthfile

```diff
@@ -0,0 +1,64 @@
+
+all:
+    BUILD +all-test
+    BUILD +all-integration-test
+    BUILD +npm
+
+all-test:
+    BUILD --build-arg ELIXIR=1.9.4  --build-arg OTP=20.3.8.26 +test
+    BUILD --build-arg ELIXIR=1.10.4 --build-arg OTP=23.1.1 +test
+    BUILD --build-arg ELIXIR=1.11.0 --build-arg OTP=21.3.8.18 +test
+    BUILD --build-arg ELIXIR=1.11.0 --build-arg OTP=23.1.1 +test
+ 
+test:
+    FROM +setup
+    COPY --dir assets config installer lib integration_test priv test ./
+    RUN mix test
+
+all-integration-test:
+    BUILD --build-arg ELIXIR=1.11.1 --build-arg OTP=21.3.8.18 +integration-test
+    BUILD --build-arg ELIXIR=1.11.1 --build-arg OTP=23.1.1 +integration-test
+
+integration-test:
+    FROM +setup
+    COPY --dir assets config installer lib integration_test priv test ./
```

Comment:
Why do we need `./` at the end? I assume COPY only copy the files in a directory so that copies the files at root?

Reply:
The last one is the destination, so this is copying all those folders into the working directory. I wasn't actually sure if they were all needed to run the integration test. Are there any that are unnecessary?